content/resources/Editor/Collisions/Advanced_Collisions.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Advanced Collisions
+image: 
+description: Understanding how to create and edit advanced collisions in a Babylon.JS Editor project
+keywords: editor, collisions
+further-reading:
+video-overview:
+video-content:
+---
+
+## Setting Advanced Collisions
+
+Checking collisions on complex objects can have a big impact on performances even for cameras and meshes
+using the `.moveWithCollisions` method. The editor provides a tool to easily setup collisions shapes on any mesh object in the scene
+in order to save performances.
+
+## Introduction
+### Basic Collisions
+By default, Babylon.JS will check for collisions using a complex algorithm that checks collisions per triangle for each mesh near the cammera
+or the mesh being moved with collisions. Sometimes (often), checking collisions on the bounding box is highly sufficient.
+
+The method would consist on:
+- Create by code a new cube mesh or sphere mesh (named `collider`)
+- Add the collider as a child of the mesh
+- Apply the bounding box properties of the mesh to the collider
+- Disable collisions on the mesh and enable collisions on the collider.
+
+In that way, the collider will always extend the transform of the mesh and camera will only check collisions on the cube or sphere collider
+instead of checking collisions on the entire mesh per triangle.
+
+### Advanced Collisions
+Let's image a more complex scenario where box and sphere collisions are not sufficient: **the stairs**. In that case, only "per triangle" collisions will allow to upstair in a natural way.
+
+In case the stairs 3d model is too complex, and in order to save performances, the solution would be to provide
+(or generate on the fly) a lower detailed 3d model for the stair.
+
+The method would consis on creating a lower level of details object of the source mesh (instead of a cube or sphere) made by an
+artist or using the [Auto-LOD Simplification Tool of Babylon.JS](https://doc.babylonjs.com/divingDeeper/mesh/simplifyingMeshes)

```

Comment:
if you want the build process to make sure this link exists don't use `https://doc.babylonjs.com`, just use the local URL. this way it will be a part of the build process' task to make sure you linked correctly

Reply:
Thanks @RaananW !
Just updated all my links in the commit :)